src/probability/independence.lean

```diff
@@ -250,11 +250,12 @@ section from_measurable_spaces_to_sets_of_sets
 /-! ### Independence of measurable space structures implies independence of generating π-systems -/
 
 lemma Indep.Indep_sets {α ι} [measurable_space α] {μ : measure α} {m : ι → measurable_space α}
-  {s : ι → set (set α)} (hms : ∀ n, m n = measurable_space.generate_from (s n))
+  {s : ι → set (set α)} (hms : ∀ n, m n = generate_from (s n))
   (h_indep : Indep m μ) :
   Indep_sets s μ :=
 begin
   refine (λ S f hfs, h_indep S (λ x hxS, _)),
+  change (m x).measurable_set' (f x),
```

Comment:
What goes wrong without this?

Reply:
`dsimp only` works too, and is probably better.

Should the statement of `measurable_set_generate_from` be expressed with `measurable_set[m] s` too?

Reply:
I'm not sure about `dsimp only`. `dsimp only` changes the goal to `f x ∈ set_of measurable_set` for which `m x` is still not visible

Reply:
Since `measurable_set_generate_from` is not a rewrite lemma and the two versions are defeq, I would think the expressing it in terms of `measurable_set` wouldn't be much different. Nonetheless, I've changed it to `measurable_set` for good measure. I can change it back if its not worth it

Reply:
>  for which m x is still not visible. 

Not visible to a human maybe, but lean can still see it!